scripts/upgrade

```diff
@@ -103,13 +93,6 @@ chown -R root: $final_path
 #=================================================
 ynh_script_progression --message="Upgrading SSOwat configuration..."
 
-# Make app public if necessary
-if [ $is_public -eq 1 ]
-then
-	# unprotected_uris allows SSO credentials to be passed anyway.
-	ynh_app_setting_set --app=$app --key=skipped_uris --value="/"
```

Comment:
you must remove the key `skipped_uris` if it exists in the `ENSURE DOWNWARD COMPATIBILITY` section